sample-app/src/main/java/com/joinforage/android/example/pos/receipts/templates/txs/TxDataLayout.kt

```diff
@@ -55,5 +55,5 @@ internal class CashWithdrawalLayout(
     override val withdrawalAmt: String
 ) : TxDataLayout() {
     override val snapAmt: String = ZERO_TX
-    override val cashAmt: String = ZERO_TX
+    override val cashAmt: String = withdrawalAmt
```

Comment:
This sample receipt from the certification script seems to suggest the `CASH` line should always have a value if the cash was touched at all. So if it was a withdrawal for $2.00, we should see `CASH 2.00` and `CS W/D 2.00` based on my understanding.

In the cash-back scenario where there's both a purchase and cash back, the `CASH` value would be the combination of the purchase and the cashback. So a $2.00 purchase with $2.00 cashback would be `CASH 4.00` and `CS W/D 2.00`, again based on my interpretation of this / understanding.

Reply:
yup I agree with this. This definitely was confusing me so I'm glad you were able to make sense of it for the rest of us 🙌